samples/dashboard/demos/climate/demo.js

```diff
@@ -72,51 +74,169 @@ async function ajaxAll(requests) {
+                                    )
+                                ));
                         }
-                        return years;
-                    })()
+                    },
+                    tooltip: {
+                        headerFormat: '',
+                        pointFormat: '{point.x:%Y-%m-%d}'
+                    }
                 }],
                 title: {
-                    text: void 0
+                    margin: 0,
+                    text: 'Timeline 2010'
                 },
                 xAxis: {
-                    visible: false
+                    tickPositions: buildDateTicks(),
+                    // tickWidth: 0,
+                    type: 'datetime',
+                    visible: true,
+                    labels: {
+                        format: '{value:%m-%d}'
+                    }
                 },
                 yAxis: {
                     visible: false
+                },
+                legend: {
+                    enabled: false
                 }
             }
         }, {
-            cell: 'kpi-1',
-            type: 'html',
-            // dimensions: {
-            //     width: 200,
-            //     height: 200
-            // },
-            elements: [{
-                tagName: 'img',
-                attributes: {
-                    src: highchartsLogo
+            cell: 'world-map',
+            type: 'Highcharts',
+            chartConstructor: 'mapChart',
+            chartOptions: {
+                chart: {
+                    map: topology,
+                    spacing: [0, 0, 0, 0],
+                },
+                colorAxis: {
+                    max: 333,
+                    maxColor: '#F93',
+                    min: 213,
+                    minColor: '#39F',
+                },
+                legend: {
+                    margin: 0,
+                },
+                mapView: {
+                    maxZoom: 1.4,
+                    padding: 0,
+                    projection: {
+                        name: 'Miller',
+                    },
+                    zoom: 1.4,
+                },
+                series: [{
+                    type: 'map',
+                    name: 'World Map',
+                }, {
+                    type: 'mapbubble',
+                    name: 'Temperature',
+                    data: climateTable
+                        .modified
+                        .getRows(undefined, undefined, ['lat', 'lon', 'TX']),
+                    keys: ['lat', 'lon', 'z'],
+                    colorKey: 'z',
+                    maxSize: 1.1,
+                    minSize: 1.1,
+                    opacity: 0.5,
+                    marker: {
+                        symbol: 'square'
+                    },
+                    tooltip: {
+                        pointFormatter: function () {
+                            const point = this;
+
+                            return [
```

Comment:
Consider if the date should be in the tooltip because there is no other indication of which date is currently selected.

Also, it might be good to select points on the timeline series at the top.

Reply:
Moved to Asana